src/Symfony/Component/Messenger/Tests/Transport/AmqpExt/AmqpSenderTest.php

```diff
@@ -25,14 +26,14 @@ class AmqpSenderTest extends TestCase
 {
     public function testItSendsTheEncodedMessage()
     {
-        $envelope = new Envelope(new DummyMessage('Oy'));
+        $envelope = (new Envelope(new DummyMessage('Oy')))->with(new RoutingKeyStamp('rk'));
```

Comment:
Can you make it another test actually? To make sure it does call `publish` with the default option, nothing else.